js-api-spec/legacy/importer.test.ts

```diff
@@ -924,3 +925,34 @@ describe('when importer returns non-string contents', () => {
     );
   });
 });
+
+it('compiles multiple nested relative imports and incorrect load path with renderSync', () => {
+  sandbox(dir => {
+    dir.write({
+      'src/index.scss': `
+        @import "./foo/_a.scss";
+        @import "./_includes.scss";
+      `,
+      'include/foo/_a.scss': '/* A */',
+      'include/foo/_b.scss': '/* B */',
+      'src/_includes.scss': `
```

Comment:
I kinda feel the error is sorta WAI because this file is not inside `./includes`